google/cloud/storage/blob.py

```diff
@@ -3873,23 +3887,38 @@ def open(
                 raise ValueError(
                     "encoding, errors and newline arguments are for text mode only"
                 )
+            if ignore_flush:
```

Comment:
@andrewsg Should these instead be `if ignore_flush is not None:`

if `ignore_flush=False` is accidentally set for reads, it would cause an AttributeError or such.

Reply:
Good point to focus on. I thought about that but decided that since False was the actual desired behavior, we didn't need to guard against it here. I don't think it would cause an AttributeError - is there a place that looks suspicious?

Reply:
I see what you mean. That's right; it wouldn't cause an error since `ignore_flush` is not passed in to the `BlobRead` constructor after all. I misread that.